Donation-platform/src/main/webapp/frontend/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
why do you have to create another one. I see a node_modules folder at the top level already